systemtest/src/main/java/io/strimzi/systemtest/utils/specific/CruiseControlUtils.java

```diff
@@ -48,6 +49,8 @@ private CruiseControlUtils() { }
         USER_TASKS
     }
```

Comment:
So these endpoints are already defined in the `cluster-operator` module. I know we decided against putting the cc client stuff in `operator-common` but preventing duplicate code like this might be a good argument for moving it. Particularly if the endpoints need to be updated in future.

Reply:
Hmm, so if you moved that code in operator-common I would be glad and totally agree with that. @tomncooper